### PR TITLE
Fix misspelling in bug name in `bugs.json` (`NestedCallataArrayAbiReencodingSizeValidation`)

### PR DESCRIPTION
**Pull request by yusufbenliii**
Misspelling fix

### CODE REVIEW

File: docs/bugs.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     },
     {
         "uid": "SOL-2022-2",
-        "name": "NestedCallataArrayAbiReencodingSizeValidation",
+        "name": "NestedCalldataArrayAbiReencodingSizeValidation",
```
Comment (cameel) [inline docs/bugs.json:63]: Note that this would have to be changed in more places. For example `bugs_by_version.json` refers to these identifiers.

**Comment by cameel** [comment]
Thanks for the fix but I'm not really sure if we want to change that at this point. It's been there for a long time (~3 years) and I think some tools may not be prepared for changes like that. For example I know that Etherscan displays these identifiers and I'm not sure how it would actually behave.

Personally I'd be for leaving it as is, but we should get more opinions. @chriseth, @axic?

**Comment by ekpyron** [comment]
> Thanks for the fix but I'm not really sure if we want to change that at this point. It's been there for a long time (~3 years) and I think some tools may not be prepared for changes like that. For example I know that Etherscan displays these identifiers and I'm not sure how it would actually behave.
> 
> Personally I'd be for leaving it as is, but we should get more opinions. @chriseth, @axic?

3 years? Hasn't this just been there since April this year?

**Comment by cameel** [comment]
Ah, you're right :) I looked at when first affected version was released rather than when we fixed it. It's actually the reencode bug we fixed not so long ago.

Well, in that case maybe it makes sense to fix it, though I still wonder if some tools might be caching these IDs?

**Comment by ekpyron** [comment]
> Ah, you're right :) I looked at when first affected version was released rather than when we fixed it. It's actually the reencode bug we fixed not so long ago.
> 
> Well, in that case maybe it makes sense to fix it, though I still wonder if some tools might be caching these IDs?

I'm not sure, maybe we should try to reach out - but generally, I'd tend towards fixing it.

**Comment by cameel** [comment]
Yeah, if it's so fresh, I'm for fixing it too. I'll ask on the tooling channel just in case.

**Review by cameel — DISMISSED** [review]

**Review by ekpyron — DISMISSED** [review]

**Comment by leonardoalt** [comment]
@cameel @ekpyron wait, I thought it was decided this wasn't going to be changed?

**Comment by ekpyron** [comment]
> @cameel @ekpyron wait, I thought it was decided this wasn't going to be changed?

We actually decided that we *want* to change this, we just wanted to ping Etherscan to see if this may cause issues for them.

**Comment by cameel** [comment]
The problem is how to reach them. I don't think their support form is the best way to do that for something like this.

**Comment by cameel** [comment]
I got a response from Etherscan guys that fixing this is fine as far as they're concerned. They don't have this hard-coded anywhere.

On the other hand - apparently that's not the only typo we have :) There's a much older one that one's better left as is.

**Review by cameel — APPROVED** [review]

**Pull request merged**